src/audio/eq_fir/eq_fir.c

```diff
@@ -311,38 +311,30 @@ static int eq_fir_setup(struct comp_data *cd, int nch)
  * End of algorithm code. Next the standard component methods.
  */
 
-static struct comp_dev *eq_fir_new(const struct comp_driver *drv,
-				   struct comp_ipc_config *config,
-				   void *spec)
+static int eq_fir_init(struct processing_module *mod)
 {
-	struct comp_dev *dev = NULL;
+	struct module_data *md = &mod->priv;
+	struct comp_dev *dev = mod->dev;
+	struct module_config *cfg = &md->cfg;
 	struct comp_data *cd = NULL;
-	struct ipc_config_process *ipc_fir = spec;
-	size_t bs = ipc_fir->size;
+	size_t bs = cfg->size;
 	int i;
 	int ret;
 
-	comp_cl_info(&comp_eq_fir, "eq_fir_new()");
+	comp_info(mod->dev, "eq_fir_init()");
```

Comment:
Can shorter `dev` be used in traces since it's already set to `mod->dev` ?

Reply:
sure, thanks